src/components/FeaturedPosts.js

```diff
@@ -0,0 +1,157 @@
+import Slider from 'react-slick';
+import {css} from '@emotion/core';
+import {map} from 'ramda';
+import {mapNodeToProps} from '~/utilities';
+import {LOCHMARA_COLOR, mq} from '~/constants';
+import Img from 'gatsby-image';
+import {ExternalLink, Text} from '~/components';
+import {Link} from 'gatsby';
+
+const settings = {
+  adaptiveHeight: true,
+  centerMode: false,
+  dots: true,
+  infinite: true,
+  speed: 750,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+  autoplay: true,
+  autoplaySpeed: 10000,
+  cssEase: 'ease',
+};
+
+// TODO: dynamically determine height
+const containerStyles = css`
+  display: block;
+  width: 100vw;
+  height: 18.75rem;
+
+  .slick-list {
+    overflow: hidden !important;
```

Comment:
I believe we should find a better selector to use instead of using `!important`. Please fine something else to use